variables.tf

```diff
@@ -0,0 +1,12 @@
+variable "aws_region" {
+  default = "eu-west-2" # London
+  type    = string
+}
+
+variable "aws_credintials" {
```

Comment:
Let's remove credentials from here and follow this https://learn.hashicorp.com/terraform/getting-started/build